### PR TITLE
go-binlog: add the field of ddl_schema_state (#156) (#157)

### PR DESCRIPTION
**Pull request by zimulala**
Cherry-pick #156 to v2.1
Add `.circleci/config.yml` to make CI work normally, besides update `generate-rust.sh` and `generate-go.sh` to make CI happy.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          command: |
+            echo 'export PATH=$HOME/.local/bin:$PATH' >> $BASH_ENV
+            curl -OL https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip
+            unzip protoc-$PROTOC_VERSION-linux-x86_64.zip -d $HOME/.local
+            rm protoc-$PROTOC_VERSION-linux-x86_64.zip
+            go get -u github.com/gogo/protobuf/protoc-gen-gofast
+            cd $GOPATH/src/github.com/gogo/protobuf
+            git checkout v0.5
+            rm $GOPATH/bin/protoc-gen-gofast
+            go get github.com/gogo/protobuf/protoc-gen-gofast
```
Comment (BusyJay) [inline .circleci/config.yml]: Does version matter?

Comment (zimulala) [reply .circleci/config.yml]: This file copy from v3.1

**Pull request merged**